pacmacs.el

```diff
@@ -331,6 +372,77 @@
     (pacmacs--track-object ghost)
     (pacmacs--step-object ghost)))
 
+(defun pacmacs--run-away-direction (game-object-1 game-object-2)
```

Comment:
I don't like these names. They aren't making it obvious which object will run from what. I'd suggest use some more appropriate terms such as `runner` and `bogey` here maybe?